frontend/src/metabase/admin/settings/components/SettingHeader/SettingHeader.jsx

```diff
@@ -1,19 +1,26 @@
 /* eslint-disable react/prop-types */
 import React from "react";
+import { jt } from "ttag";
 import { SettingDescription, SettingTitle } from "./SettingHeader.styled";
 
-const SettingHeader = ({ id, setting }) => (
-  <div>
-    <SettingTitle htmlFor={id}>{setting.display_name}</SettingTitle>
-    <SettingDescription>
-      {setting.warningMessage && (
-        <React.Fragment>
-          <strong>{setting.warningMessage}</strong>{" "}
-        </React.Fragment>
-      )}
-      {setting.description}
-      {setting.note && <div>{setting.note}</div>}
-    </SettingDescription>
-  </div>
-);
+const SettingHeader = ({ id, setting }) => {
+  // Disable translation, because it's needed
+  // only for whitelabel name replacement.
+  // See metabase#17043
+  /* disable ttag */
+  return (
+    <div>
+      <SettingTitle htmlFor={id}>{setting.display_name}</SettingTitle>
+      <SettingDescription>
+        {setting.warningMessage && (
+          <React.Fragment>
+            <strong>{jt`${setting.warningMessage}`}</strong>{" "}
```

Comment:
Something in `setting` is probably a JSX, that's why when I used just `t` these tests failed.
- https://app.circleci.com/pipelines/github/metabase/metabase/33280/workflows/af1c70f4-a5f2-43f9-a952-1bf40ae1bf47/jobs/1715260
- https://app.circleci.com/pipelines/github/metabase/metabase/33280/workflows/af1c70f4-a5f2-43f9-a952-1bf40ae1bf47/jobs/1715201